source/3_Netplugin.md

```diff
@@ -0,0 +1,180 @@
+# Netplugin
+
+## About
+Contiv netplugin is a generic networking plugin that is designed to provide multi host policy based networking for containerized applications.
+
+Netplugin is designed to be used with docker containers and with cluster schedulers like Swarm, Kubernetes and Mesos
+
+## Getting started
+This section will walk you through how to try netplugin in a vagrant environment
+### Prerequisits
+- VirtualBox 5.0.2 or greater
+- Vagrant 1.7.4
+- Go 1.5.0
+
+### Quick start guide
+Clone the repo and bring up the VMs
+```
+$ git clone https://github.com/contiv/netplugin
+$ cd netplugin; make demo
+$ vagrant ssh netplugin-node1
+```
+This brings up a cluster of two VMs running docker and netplugin
+
+#### Step 1: Create a network
+```
+netplugin-node1$ netctl net create contiv-net --subnet=20.1.1.0/24 --gateway=20.1.1.254 --pktTag=200
```

Comment:
--pktTag should probably be --pkt-tag

   --pkt-tag, -p 		Packet tag (Vlan/Vxlan ids)- REQUIRED